javascript/index.js

```diff
@@ -14,7 +14,11 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
```

Comment:
no need to add more addEventListener in this functions. you only need them once for each button, and those are at the end of the code. This functions will not proceed to update the DOM. check the solutions on how these work up to this point.